hw2/03-count.js

```diff
@@ -1,3 +1,20 @@
 // Add your code here
+const input = document.querySelector('input');
+const originalText = `It was the best of times, it was the worst of times, it was the age of
+wisdom, it was the age of foolishness, it was the epoch of belief, it
+was the epoch of incredulity, it was the season of Light, it was the
+season of Darkness, it was the spring of hope, it was the winter of
+despair, we had everything before us, we had nothing before us, we were
+all going direct to Heaven, we were all going direct the other way—in
+short, the period was so far like the present period, that some of its
+noisiest authorities insisted on its being received, for good or for
+evil, in the superlative degree of comparison only.`;
```

Comment:
Hardcoding long text like this is not good practice and it takes up unnecessary space. If the text in the html changed, then the programmer would have to also remember to update this file. 

Instead you can just use textContent/innerText to get the text from an html element.